src/LibraryManager.Build/Contracts/Logger.cs

```diff
@@ -21,16 +21,16 @@ public void Clear()
             Errors.Clear();
         }
 
-        public void Log(string message, Contracts.LogLevel level)
+        public void Log(string message, LibraryManager.Contracts.LogLevel level)
```

Comment:
It would feel a little cleaner if we added a using for the LogLevel class instead of having to always reference it 3 levels (`LibraryManager.Contracts.LogLevel` is pretty verbose)

Reply:
Done.